android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -195,6 +197,7 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             if (currentActivity != null) {
                 currentActivityHolder.setActivity(currentActivity);
             }
+            serializer = options.getSerializer();
```

Comment:
H: This would not work if [manual init](https://docs.sentry.io/platforms/react-native/manual-setup/native-init/) is not enabled because we won't have the options ref.
An option is to read the serializer from the HubAdapter `HubAdapter.getInstance().getOptions().getSerializer()`.